Source/MediaStorageAndFileFormat/gdcmJPEG2000Codec.h

```diff
@@ -56,6 +56,10 @@ friend class ImageRegionReader;
 
   void SetNumberOfResolutions(unsigned int nres);
 
+  /// Set Number of threads
+  /// @param nThreads : number of threads for codec, if negative value is set determine how many virtual threads are available
+  void SetNumberOfThreads(int nThreads);
```

Comment:
@pierreldff Can you update the documentation for the case `0`, it should be clear to the user that this means current thread (no new thread created if I understand correctly).